packages/react-components/lib/robots/robot-panel.tsx

```diff
@@ -32,12 +32,14 @@ export interface RobotPanelProps extends React.HTMLProps<HTMLDivElement> {
   paginationOptions?: Omit<React.ComponentPropsWithoutRef<typeof TablePagination>, 'component'>;
   verboseRobots: VerboseRobot[];
   fetchVerboseRobots: () => Promise<VerboseRobot[]>;
+  onRobotZoom?: (robot: VerboseRobot) => void;
 }
 
 export function RobotPanel({
   paginationOptions,
   verboseRobots,
   fetchVerboseRobots,
+  onRobotZoom,
   ...divProps
 }: RobotPanelProps): JSX.Element {
```

Comment:
Just some legacy problems, in keeping the spirit of re-usable components, I think `fetchVerboseRobots` should be replaced with `onRefresh`, and leave it to the parent component to handle the refresh logic.

For now, you can just put a FIXME to refactor it in the future.